lib/ansible/module_utils/k8s/raw.py

```diff
@@ -175,7 +175,13 @@ def perform_action(self, resource, definition):
             # Differences exist between the existing obj and requested params
             if not self.check_mode:
                 try:
-                    k8s_obj = resource.patch(definition, name=name, namespace=namespace).to_dict()
+                    params = dict(name=name, namespace=namespace)
+                    if self.params['merge_type']:
+                        from distutils.version import LooseVersion
+                        if LooseVersion(self.openshift_version) < LooseVersion("0.6.1"):
```

Comment:
also change this to 0.6.2, just came to my attention that I made an error in the release process and the 0.6.1 release didn't update the openshift.__version__. 0.6.2 is identical to 0.6.1 except with the version properly set. Sorry for the inconvenience